### ISSUE TITLE
KeyError raised when removing or renaming an existing model

### ISSUE THREAD

**Issue by licongy** [tortoise/aerich#386]
KeyError raised when removing/renaming an existing model,  regardless of whether a relationship field exists for that model.

for example, remove a simple model:
```
class Test(models.Model):
    title = fields.CharField(max_length=256, default="", null=False)
```

or rename above model to:
```
class TestNew(models.Model):
    title = fields.CharField(max_length=256, default="", null=False)
```

will raise:
```
Traceback (most recent call last):
  File ".../bin/aerich", line 8, in <module>
    sys.exit(main())
             ~~~~^^
  File ".../lib/python3.13/site-packages/aerich/cli.py", line 247, in main
    cli()
    ~~~^^
  File ".../lib/python3.13/site-packages/asyncclick/core.py", line 1205, in __call__
    return anyio.run(self._main, main, args, kwargs, **opts)
           ~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File ".../lib/python3.13/site-packages/anyio/_core/_eventloop.py", line 74, in run
    return async_backend.run(func, args, {}, backend_options)
           ~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File ".../lib/python3.13/site-packages/anyio/_backends/_asyncio.py", line 2347, in run
    return runner.run(wrapper())
           ~~~~~~~~~~^^^^^^^^^^^
  File "/opt/homebrew/Cellar/python@3.13/3.13.0_1/Frameworks/Python.framework/Versions/3.13/lib/python3.13/asyncio/runners.py", line 118, in run
    return self._loop.run_until_complete(task)
           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^
  File "/opt/homebrew/Cellar/python@3.13/3.13.0_1/Frameworks/Python.framework/Versions/3.13/lib/python3.13/asyncio/base_events.py", line 721, in run_until_complete
    return future.result()
           ~~~~~~~~~~~~~^^
  File ".../lib/python3.13/site-packages/anyio/_backends/_asyncio.py", line 2335, in wrapper
    return await func(*args)
           ^^^^^^^^^^^^^^^^^
  File ".../lib/python3.13/site-packages/asyncclick/core.py", line 1208, in _main
    return await main(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File ".../lib/python3.13/site-packages/asyncclick/core.py", line 1120, in main
    rv = await self.invoke(ctx)
         ^^^^^^^^^^^^^^^^^^^^^^
  File ".../lib/python3.13/site-packages/asyncclick/core.py", line 1739, in invoke
    return await _process_result(await sub_ctx.command.invoke(sub_ctx))
                                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File ".../lib/python3.13/site-packages/asyncclick/core.py", line 1485, in invoke
    return await ctx.invoke(self.callback, **ctx.params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File ".../lib/python3.13/site-packages/asyncclick/core.py", line 824, in invoke
    rv = await rv
         ^^^^^^^^
  File ".../lib/python3.13/site-packages/aerich/cli.py", line 75, in migrate
    ret = await command.migrate(name, empty)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File ".../lib/python3.13/site-packages/aerich/__init__.py", line 130, in migrate
    return await Migrate.migrate(name, empty)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File ".../lib/python3.13/site-packages/aerich/migrate.py", line 152, in migrate
    cls.diff_models(new_version_content, last_version, False)
    ~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File ".../lib/python3.13/site-packages/aerich/migrate.py", line 242, in diff_models
    model = cls._get_model(new_model_describe["name"].split(".")[1])
  File ".../lib/python3.13/site-packages/aerich/migrate.py", line 72, in _get_model
    return Tortoise.apps[cls.app][model]
           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^
KeyError: 'Test'
```

Version:
aerich : 0.8.1 6270c47
tortoise-orm : 0.22.2
asyncpg : 0.30.0
postgre : 16.4
python : 3.13

**Comment by waketzheng** [comment]
@licongy KeyError fixed by [56da0e7](https://github.com/tortoise/aerich/commit/56da0e7e3c6329f122dfc46e37a15774e5b7f6f8)

But I find a new bug: downgrade does not create the old table
```
(venv) mac10.12 in ~/trying/something λ cat migrations/models/1*
from tortoise import BaseDBAsyncClient


async def upgrade(db: BaseDBAsyncClient) -> str:
    return """
        CREATE TABLE IF NOT EXISTS "newtest" (
    "id" SERIAL NOT NULL PRIMARY KEY,
    "title" VARCHAR(256) NOT NULL  DEFAULT ''
);
        DROP TABLE IF EXISTS "test";"""


async def downgrade(db: BaseDBAsyncClient) -> str:
    return """
        DROP TABLE IF EXISTS "newtest";"""
```
I'll see it later.

BTW: if you want to rename a table, you should do it like this:
```py
class Test(Model):
    class Meta:
        table = 'newtest'
```

**Comment by licongy** [comment]
@waketzheng 
Thank you very much for the quick fix and giving the tip :)

I am used to explicitly defining the table name in the Meta, and modification of this table name works fine so I didn't mention in the above.

However, after modifying the table name, that will be great to do a CONSTRAINT name update of the relationship field along with it. This may not that important, but if the CONSTRAINT name is out of the table name too much, it may cause trouble for some people who need fine-grained database management.
